tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/bluetooth.py

```diff
@@ -34,16 +33,18 @@ def device_name(self):
 
     def enable_bluetooth(self):
         self.marionette.find_element(*self._bluetooth_label_locator).tap()
```

Comment:
This was the problem, the locator is no longer valid, and currently there is no way to check whether the checkbox is checked, otherwise I'd simply check its value.  If I'm wrong, please let me know.